eth/gen_config.go

```diff
@@ -21,6 +21,7 @@ func (c Config) MarshalTOML() (interface{}, error) {
 		NetworkId               uint64
 		SyncMode                downloader.SyncMode
 		EthDiscoveryURLs        []string
+		SnapDiscoveryURLs       []string
```

Comment:
Is this field added here by mistake? (if you really wanted to add it, it's fine, no need open another PR just for this change)